.github/CHANGELOG.md

```diff
@@ -23,3 +23,4 @@
 9. [ND] Add DME distances, VOR/ADF needles and functioning ADF2 - @blitzcaster (bltzcstr)
 10. [OVHD] Fixed Battery Indicator Colour - @nathaninnes (Nathan Innes)
 11. [MISC] Removed Fuel Patch from MSFS Update 1.8.3 - @nathaninnes (Nathan Innes)
+12. [OVHD] Implemented missing functionality for multiple overhead buttons and switches - @wpine215 (Iceman)
```

Comment:
Change to current month

Reply:
Fixed.

Reply:
Can you please merge master and update the changelog location again, preview doesn't look right.